docs/rackhd/ubuntu_source_installation.rst

```diff
@@ -23,12 +23,14 @@ You may need to update this and /etc/dhcpd.conf to match your local network
 configuration.
 
 This will install all the relevant dependencies and code into ~/src, expecting
-that it will be run with node-foreman.
+that it will be run with `pm2`_.
```

Comment:
Suggest to add some frequently used command here, so user will feel easier to use it. some example below:
- sudo pm2 restart all: restart all RackHD services
- sudo pm2 restart on-taskgraph: restart the on-taskgraph service only.
- sudo pm2 logs: show the combined real-time log for all RackHD services
- sudo pm2 logs on-taskgraph: show the on-taskgraph real-time log
- sudo pm2 logs flush: clean the RackHD logs
- sudo pm2 status: Show the status of RackHD services